cfme/utils/log_validator.py

```diff
@@ -81,3 +83,19 @@ def _verify_match_logs(self):
         for pattern in self.matched_patterns:
             if pattern not in self.matches:
                 pytest.fail('Expected pattern {} did not match'.format(pattern))
+
+    def wait_for_log_validation(self, **kwargs):
+        """ Wait for log validation, takes the kwargs as wait_for. This function will reduce
+            duplicate functions in tests that wait_for log_validation. It is necessary to create
+            this function since _verify_match_logs raise pytest.fail() when it fails to find the
+            match pattern in the logs.
+
+            Note that you must call fix_before_start() before making use of this function.
+        """
+        def validate():
+            try:
+                self.validate_logs()
+                return True
+            except Failed:
+                return False
+        wait_for(validate, **kwargs)
```

Comment:
**Required** What do you think about passing a default `delay`, `num_sec`, and a `message`?